### ISSUE TITLE
MergeMarkups fails in merging the LM sets, if one node has the Number of control points locked. 

### ISSUE THREAD

**Issue by muratmaga** [SlicerMorph/SlicerMorph#345]
**Describe the bug**
Create two landmark set. In one of them lock the  number of control points property. Try to merge the these two sets, and notice that Merge Markups throws this error:

```
AddNControlPoints: Markup node control point number is locked.

AddNControlPoints: Markup node control point number is locked.

AddNControlPoints: Markup node control point number is locked.

AddNControlPoints: Markup node control point number is locked.
```

When merging, this property should be ignored. Requiring the user manually unlock these is not feasible (e.g., if they have done the landmarking using a landmark template, for every file this property will be locked).

**Comment by smrolfe** [comment]
This bug was fixed in the commit: https://github.com/SlicerMorph/SlicerMorph/commit/0ec34cb0a5440972d8f0be2b97ec68a2939511be

**Comment by muratmaga** [comment]
Is it available on stable though? I still have it with the extension installed on Stable.

**Comment by smrolfe** [comment]
Yes, on the stable branch the fix was committed here: 7e4c9e736428abbc81d089873732eb53a97826a8
